src/sensors/mpu9250sensor.cpp

```diff
@@ -168,14 +168,12 @@ void MPU9250Sensor::getMPUScaled()
     float temp[3];
     int i;
     int16_t ax,ay,az,gx,gy,gz,mx,my,mz;
```

Comment:
You could move these variables (`ax,ay,az,gx,gy,gz,mx,my,mz`) into the `if defined(_MAHONY_H_) || defined(_MADGWICK_H_)` to save even more stack, I believe.

Reply:
oh, I fixed it in my another code. I missed this code. thanks.